magento/magento2
# Fix issue button used action-primary class changed color if click outside

### Description (*)

Continue works by @neerajchand after long inactivity in PR 27560
update with correction in code style base magento standards

### Related Pull Requests


### Fixed Issues (if relevant)

1. fixes magento/magento2#27095: In Floating cart delete pop-up, In Floating cart delete pop-up, When user click

### Manual testing scenarios (*)

1. Magento 2.3.4+
2. Theme luma
3. Add any products to cart
4. Remove product added
5. Click outside of modal confirm remove

### Questions or comments


### Contribution checklist (*)
